Makefile

```diff
@@ -32,14 +32,11 @@ push:
 fmt:
 	@gofmt -l -w $(SRCS)
 
-test: vet fmtcheck spdxcheck lint
+test: vet fmtcheck spdxcheck
 
 vet:
 	go vet ./...
 
-lint:
-	@hack/check_golangci-lint.sh
-
```

Comment:
I'd prefer to keep the shell script for now and just remove it from `make all` via `make test` in line 35.

OK for me if it is broken now, I would like to fix it in a follow-up PR then.

The reason for this: I think it is comfortable to have a working command which replicates the ci command and version. We can't enforce someones IDE's :-)